apps/marketing/content/blog/switching-from-slack-to-discord

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
There are currently some issues with this file which is causing it to not render correctly locally:

1. This file needs the `.mdx` file extension
2. Having tabs before lists is causing the following issue:

```
     • "blog/switching-from-slack-to-discord.mdx" failed with YAMLParseError: Tabs are not allowed as indentation at line 10, column 1:
     
       - Announcement
        - Community
     ^
```

The solution would be to change the tab before `  - Community` on line 10 to two spaces.

Reply:
You can paste this file into something like this which will help format it:

https://codebeautify.org/mdx-formatter

Reply:
my bad! also fixed